### PR TITLE
TASK: Add configurable rsync params

### PR DESCRIPTION
**Pull request by mstruebing**

### CODE REVIEW

**Comment by mstruebing** [comment]
@grebaldi @mficzel

File: Classes/Sitegeist/MagicWand/Command/CloneCommandController.php [hunk 1]
```diff
@@ -303,4 +304,16 @@ protected function checkConfiguration($remotePersistenceConfiguration)
...
         $this->outputLine(' - Configuration seems ok ...');
     }
+
+    /**
+     * @return string
+     */
+    protected function getRsyncParams()
+    {
+        foreach ($this->rsyncParams as $index => $param) {
+            index == 0 ? $rsyncParams = $param : $rsyncParams .= ' ' . $param;
```
Comment (grebaldi) [inline Classes/Sitegeist/MagicWand/Command/CloneCommandController.php]: here's a missing `$` in front of `index`.

File: Classes/Sitegeist/MagicWand/Command/CloneCommandController.php [hunk 2]
```diff
@@ -303,4 +304,16 @@ protected function checkConfiguration($remotePersistenceConfiguration)
         }
         $this->outputLine(' - Configuration seems ok ...');
     }
+
+    /**
+     * @return string
+     */
+    protected function getRsyncParams()
+    {
+        foreach ($this->rsyncParams as $index => $param) {
```
Comment (grebaldi) [inline Classes/Sitegeist/MagicWand/Command/CloneCommandController.php]: Where is `$this->rsyncParams` coming from? Did you forget to `@InjectConfiguration` it?

File: Classes/Sitegeist/MagicWand/Command/CloneCommandController.php [hunk 3]
```diff
@@ -303,4 +304,16 @@ protected function checkConfiguration($remotePersistenceConfiguration)
         }
         $this->outputLine(' - Configuration seems ok ...');
     }
+
+    /**
+     * @return string
+     */
+    protected function getRsyncParams()
```
Comment (grebaldi) [inline Classes/Sitegeist/MagicWand/Command/CloneCommandController.php]: I'd rather go for `renderRsyncParams` tbh, but that's just nitpicking.

**Comment by grebaldi** [comment]
Oh, I forgot something: It would be great, if you'd squashed those duplicate commits.

**Pull request closed without merging**